server/server_test.go

```diff
@@ -4834,30 +4830,10 @@ func TestDistributedTracingEnabled(t *testing.T) {
 		}}`)
 
 	ctx := context.Background()
-	_, tracerProvider, err := distributedtracing.Init(ctx, c, "foo")
+	_, _, err := distributedtracing.Init(ctx, c, "foo")
```

Comment:
`otelhttp.Handler` type was removed in https://github.com/open-telemetry/opentelemetry-go-contrib/pull/2964. See [this](https://github.com/open-telemetry/opentelemetry-go-contrib/pull/2964#discussion_r1177591249) comment for details.